code/modules/power/supermatter/supermatter.dm

```diff
@@ -695,6 +794,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		default_unfasten_wrench(user, tool, time = 20)
 	return TRUE
 
+/obj/machinery/power/supermatter_crystal/emag_act(obj/item/W, mob/living/user,)
+
+
```

Comment:
Whats this used for?

Reply:
Weird, I left that one out before, might have gotten readded when I was shifting around braches due to github desktop fuckery. I need to learn a proper CLI